### PR TITLE
[bitnami/clickhouse] add change notes to use Kubernetes 1.8

### LINKED ISSUE #13855: [bitnami/clickhouse] install failed
### Name and Version

bitnami/clickhouse 2.0.0

### What steps will reproduce the bug?

helm install clickhouse, throw err
`Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(StatefulSet.spec.template.spec.securityContext): unknown field "seccompProfile" in io.k8s.api.core.v1.PodSecurityContext`

kube version: v1.18.16

### Are you using any custom parameters or values?

no

### What is the expected behavior?

_No response_

### What do you see instead?

delete seccompProfile in values is ok
image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lusson-luo**
Signed-off-by: lusson <lusson@foxmail.com>

### Description of the change

add change notes in readme and values.yaml to use Kubernetes 1.8

### Benefits

Install clickhouse is failed when i use Kubernetes 1.8

### Possible drawbacks

no

### Applicable issues

  - fixes #13855

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by CeliaGMqrz** [comment]
Hi @lusson-luo, 

Thanks for your contribution! 

Can you change the title as described in our checklist?
> Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title

File: bitnami/clickhouse/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: clickhouse
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/clickhouse
   - https://github.com/ClickHouse/ClickHouse
-version: 2.1.1
+version: 2.1.3
```
[the remark this answers is not in the document]
Comment (lusson-luo) [reply bitnami/clickhouse/Chart.yaml]: ok, i solve it

**Comment by lusson-luo** [comment]
> Hi @lusson-luo,
> 
> Thanks for your contribution!
> 
> Can you change the title as described in our checklist?
> 
> > Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title

ok, i solve it

File: bitnami/clickhouse/README.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 - PV provisioner support in the underlying infrastructure
 - ReadWriteMany volumes for deployment scaling
 
+> **Tip**: If use Kubernetes 1.18, need commented follow code in values
```
Comment (CeliaGMqrz) [inline bitnami/clickhouse/README.md:35]: Thanks for your contribution. In order to follow a similar approach to other READMEs in the repository, I proposed some minor changes.
```suggestion
> If you are using Kubernetes 1.18, the following code needs to be commented out.
```

File: bitnami/clickhouse/values.yaml [hunk 3]
```diff
@@ -190,7 +190,7 @@ resources:
 ## @param podSecurityContext.enabled Enabled ClickHouse pods' Security Context
 ## @param podSecurityContext.fsGroup Set ClickHouse pod's Security Context fsGroup
 ## @param podSecurityContext.seccompProfile.type Set ClickHouse container's Security Context seccomp profile
-##
+## If use Kubernetes 1.18, need commented podSecurityContext.seccompProfile
```
Comment (CeliaGMqrz) [inline bitnami/clickhouse/values.yaml:193]:
```suggestion
## If you are using Kubernetes 1.18, the following code needs to be commented out.
##
```

**Review by CeliaGMqrz — COMMENTED** [review]
Thanks for your contribution. In order to follow a similar approach to other READMEs in the repository, I proposed some minor changes.

**Review by carrodher — APPROVED** [review]

**Pull request merged**